src/foundryvtt-gmScreen.ts

```diff
@@ -74,6 +74,11 @@ Handlebars.registerHelper(`${MODULE_ABBREV}-enrich`, function (str) {
   return TextEditor.enrichHTML(str);
 });
 
+Handlebars.registerHelper(`${MODULE_ABBREV}-journal`, function (uuid) {

```

Comment:
I don't think we'll need this handler.

Reply:
Thanks, will take a look at the lookup approach.